spec/pagerduty_spec.rb

```diff
@@ -229,17 +238,19 @@
 
         context "PagerDuty fails to create the incident" do
           Given {
-            transport.stub(:send_payload => {
-              "status" => "failure",
-              "message" => "Event not resolved",
-            })
+            allow(transport).to receive(:send_payload).and_return(
+              {
+                "status" => "failure",
+              "message" => "Event not resolved"
+              }
```

Comment:
These `{ }` aren't required here. IMO the code reads better without them. (minor indentation issue here also.)
```ruby
 allow(transport).to receive(:send_payload).and_return(
  "status" => "failure",
  "message" => "Event not resolved",
)
```
Similar comment for above changes.